application/src/main/scala/com/azavea/franklin/api/Server.scala

```diff
@@ -25,8 +28,22 @@ import sttp.tapir.docs.openapi._
 import sttp.tapir.openapi.circe.yaml._
 import sttp.tapir.swagger.http4s.SwaggerHttp4s
 
+import scala.concurrent.ExecutionContext
+
+import java.util.concurrent.Executors
+
 object Server extends IOApp {
 
+  val franklinIO: ContextShift[IO] = IO.contextShift(
```

Comment:
This was brought over from `backsplash` in an attempt to try and speed things up -- not sure if it helped though.